Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -316,6 +298,9 @@ const SubmissionDetails = () => {
                 <Heading
                   type={HeadingType.secondary}
                   className={styles.taskHeading}
+                  style={problemNameHeadingText.length >= 30
+                      ? { marginBottom: 0 }
+                      : { marginBottom: '24px' }}
```

Comment:
Move this to scss?

Reply:
it's not in a scss file because it's a conditional style, the other way around would be to append classname, that would have just one style rule, that's why I used it as an inline - not to have class names that have only one rule inside of them. I can move it to a class name, but since it would be just one rule inside, I preffered to keep it simpler